docs/guide/epuck.md

```diff
@@ -291,6 +291,9 @@ A condition can be set on this value to get a line follower behavior.
 An example of BotStudio can be found by opening the "WEBOTS\_HOME/projects/robots/gctronic/e-puck/world/e-puck\_botstudio.wbt" world file (see below).
 The BotStudio window can be opened by selecting the "Show Robot Window" item in the e-puck [context menu](the-3d-window.md#context-menu) when the controller points on a *.bsg* file.
 
+BotStudio needs some additional Qt libraries to run, so it is necessary to add a [`runtime.ini` file](controller-programming.md#environment-variables) in the folder containing the BotStudio controller file.
+A sample `runtime.ini` specifiying the required libraries could be found at ""WEBOTS\_HOME/projects/robots/gctronic/e-puck/controllers/obstacle/runtime.ini".
```

Comment:
It would be better to use a hyperlink here to refer to the `runtime.ini` sample file.
```suggestion
A sample `runtime.ini` specifiying the required libraries could be found at "WEBOTS\_HOME/projects/robots/gctronic/e-puck/controllers/obstacle/runtime.ini".
```

Reply:
Done:
https://www.cyberbotics.com/doc/guide/epuck?version=fix-botsudio-robot-window